pkg/networkoptions/network_options.go

```diff
@@ -334,6 +335,11 @@ func GetNetworkFromCmdLineFlags(
 		}
 	}
 
+	if networkFlags.Endpoint != "" {
+		re := regexp.MustCompile(`/+$`)
```

Comment:
why do you need `regex` for something that can be done via `strings.TrimSuffix(s, "/")`

Reply:
I want to remove any amount of trailing / just in case

Reply:
in my opinion this is excessive but approved it anyway